src/style.css

```diff
@@ -2,0 +1,46 @@
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  margin-bottom: 0;
+}
+
+.star {
+  width: 24px;
+  height: 24px;
+  background-image: url('./images/star.svg');
+  background-size: cover;
+  margin-right: 8px;
+}
+
+.stars--0 .star:nth-child(-n + 0) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--1 .star:nth-child(-n + 1) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--2 .star:nth-child(-n + 2) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--3 .star:nth-child(-n + 3) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--4 .star:nth-child(-n + 4) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--5 .star:nth-child(-n + 5) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars .star:last-child {
+  margin-right: 0;
```

Comment:
Good job on removing the margin on the last child to maintain the layout consistency without extra spacing at the end.